extension/content/personas.css

```diff
@@ -90,6 +95,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
```

Comment:
Actually I coded this 1 month ago, just waited the previous one to be approved. And I do not recall why I did not add the -moz-image-region.

But actually the personas_20x20.png image is already 20x20 so there is no need to place a  -moz-image-region property here.

Also because there is only 1 **toolbar[iconsize="small"]**  rule in this css file and this is specific to SeaMonkey Small Toolbar Icon state, the -moz-image-region is already in its inherited default value. So it seems there is no need for an extra -moz-image-region.